AccessKit/accesskit
# fix: Bump objc2 to 0.5.0; bring icrate 0.1.0

Replaces #200

- [x] https://github.com/madsmtm/objc2/issues/537,
- [x] https://github.com/rust-windowing/winit/pull/3256

- Update to latest objc2,
- Introduce icrate as a dependency,
- Remove handmade AppKit types in favor of auto-generated ones provided by icrate,
- Use type aliases from icrate as much as possible

The compile time of accesskit_macos increases a bit due to icrate. This shouldn't be a big issue for Rust users who will probably have it in their dependency chain already. Bindings generation will be slightly slower, but I don't think this should be a concern.
